internal/pkg/deploy/deploy.go

```diff
@@ -4,6 +4,26 @@
 // Package deploy holds the structures to deploy infrastructure resources.
 package deploy
 
+import (
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws/arn"
+	rg "github.com/aws/copilot-cli/internal/pkg/aws/resourcegroups"
+	"github.com/aws/copilot-cli/internal/pkg/aws/session"
+	"github.com/aws/copilot-cli/internal/pkg/config"
+)
+
+const (
+	ecsServiceResourceType = "AWS::ECS::Service"
+	// AppTagKey is tag key for Copilot app.
+	AppTagKey = "copilot-application"
+	// EnvTagKey is tag key for Copilot env.
+	EnvTagKey = "copilot-environment"
+	// ServiceTagKey is tag key for Copilot svc.
+	ServiceTagKey = "copilot-service"
```

Comment:
nit: can we move these consts to a separate group since they're all about tag keys whereas `ecsServiceResourceType` is unrelated.